packages/frontend/src/components/connect-button/account-menu/index.tsx

```diff
@@ -104,7 +103,8 @@ export function AccountMenu({
                                 <SafeLogo className={styles.safeLogo} />
                             </div>
                         ) : (
-                            <Image
+                            // eslint-disable-next-line @next/next/no-img-element
+                            <img
```

Comment:
Why not `Image` here?

Reply:
Addressed in 53c17b11315e89fa52df2003abd78b47bff65fbb.